### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by nerdpiece** [Whisky-App/Whisky#475]
### Description

Controllers detected by Whisky and Steam but not games.

### Steps to reproduce

Step 1: connect controller

step 2: open steam

step 3 launch game

step 4 get mad

### Expected behaviour

the controller working

### Logs

```bash
Whisky Version: 2.0.1
Date: 10/4/2023, 1:00:01 PM
macOS Version: Version 14.0 (Build 23A344)
Bottle Name: Steam
Wine Version: 7.7.0
Windows Version: win10
Bottle URL: /Users/JeffreyL/Library/Containers/com.isaacmarovitz.Whisky/Bottles/14632F63-E42F-46C1-978C-EE1D15915473

Arguments: regedit 

Launched Wine (7347)
Process exited with code 0
```

### What version of Whisky are you using?

2.0.1

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by DmitryShapovalov** [comment]
The same issue happened to me.

**Comment by govind-maheshwari2** [comment]
Same issue I am facing and yes game support controller

**Comment by Emberr** [comment]
same issue, steam and whiskey both detect DS4 controller but Elden Ring does not work. (when ps button is held down only camera rotation works + rumble)

**Comment by HalfWhitt** [comment]
Yes, same problem here (in this case, with Risk of Rain 2013). Steam sees that my controller is connected, and correctly displays input on the test screen. However, Risk of Rain is completely unresponsive to controller input, despite supporting it.

Even if I turn on Steam Control and tell Steam to remap all my controller buttons to keyboard input, the game still receives none of it.

**Comment by s4m2d** [comment]
I geh the Same Isabel in Whiskey 2.2.0
Running on Mac OS 14 and 14.1

**Comment by askimet** [comment]
Have the same issue, MacOS 14.1.1, recognizes my controller, so does Whiskey 2.2.1 and Steam. But When I start the game (LEGO Star Wars the Skywalker Saga) , it doesn't react at all.

**Comment by BeeGodwin** [comment]
+1 here with Baldur's Gate 3. Disabling Steam Input doesn't make a difference in my case.

**Comment by Spirarel** [comment]
> +1 here with Baldur's Gate 3. Disabling Steam Input doesn't make a difference in my case.

Doesn't Baldur's Gate 3 run natively? You shouldn't need Whisky for this.

**Comment by BeeGodwin** [comment]
> > +1 here with Baldur's Gate 3. Disabling Steam Input doesn't make a difference in my case.
> 
> Doesn't Baldur's Gate 3 run natively? You shouldn't need Whisky for this.

It did, but since updating to Sonoma it randomly hangs running natively, so I was looking for an alternative. 

Also, just because there's a way to work around a bug, doesn't mean it's resolved...

**Comment by Spirarel** [comment]
If you read the entire history of the issue, you would see I that I haven't suggested anywhere that this is resolved. In fact, I find this bug super annoying. I was simply offering a solution to _your_ particularly problem.

**Comment by cipitaua** [comment]
Hello, I'm encountering the same issue with "It takes two", using either GEproton or wine-staging on debian, by directly running wine without frontends like whisky. Thus I don't think the problem is whisky-related.

**Comment by vvuk** [comment]
Just posted on something similar here: https://github.com/orgs/Whisky-App/discussions/833 . Try forcing the input libraries in Wine configuration to "builtin" (you have to manually add dinput8 if the game uses it).

**Comment by cipitaua** [comment]
https://selfmadepenguin.wordpress.com/2024/02/14/how-i-solved-my-gamecontroller-problems/

**Comment by Space-Cadet0** [comment]
There seems to be a bug when using D3DMetal GPTK and controllers via steam for some (but not all) games.  Swithcing to DXVK resolves the issue, but you lose the D3DMetal GPTK performance benefits.

https://github.com/Whisky-App/Whisky/issues/844

**Comment by ekl3m** [comment]
I am experiencing the same issue with my favorite controller-played game - The Binding of Isaac : Repentance.

It sadly just crashes without opening even for a millisecond. The moment I disable Bluetooth connection with my controller (dualsense 5) it just launches without any problems.

Hope to find a fix soon, I don't think these libraries shown earlier do the job for me, tried it. Maybe an official statement how to fix this?

**Comment by Space-Cadet0** [comment]
I don't think that will work, there a number of other mappers that I have tried that don't fully solve the issue.
These issues are specific to MacOS and mostly D3DMetal in GPTK

**Comment by ekl3m** [comment]
True, I'm playing on MacOS Sonoma. As I can see, this mapper is for Linux, mostly Debian but that was just a quick Google search.

**Comment by cipitaua** [comment]
sorry I did not realize you were talking about MacOS. Then I don't know.
